src/vstart.sh

```diff
@@ -948,6 +946,9 @@ EOF
                 ceph_adm fs flag set enable_multiple true --yes-i-really-mean-it
             fi
 
+	    # wait for volume module to load
```

Comment:
i am surprised that the editor variables didn't kick in. could you please use spaces for the indent?